16S/data_construction/prior_16S_bahram/3._trimming_samples.r

```diff
@@ -0,0 +1,60 @@
+#trimming primers from fastq files for bahram 2018.
```

Comment:
According to Git, this file is completely new, but it sure looks a lot like the deleted file 2._trimming_samples.r. My guess is that you manually renamed the file, and then added it and deleted the other, rather than asking git itself to move (i.e. `git mv`). You'll want to switch to git mv in the future as it (A) retains file revision history [manual move breaks the versioning] and (B) prevents file bloat